cmd/validation.go

```diff
@@ -67,3 +67,14 @@ func coalesceWorkspace() (string, error) {
 
 	return "", errors.New("no valid workspace source found")
 }
+
+func validateRole(role string) error {
+	validRoles := []string{"WORKSPACE_ADMIN", "WORKSPACE_EDITOR", "WORKSPACE_VIEWER"}
+
+	for _, validRole := range validRoles {
+		if role == validRole {
+			return nil
+		}
+	}
+	return errors.Errorf("please use one of: %s", strings.Join(validRoles, ", "))
```

Comment:
@schnie Please take a look on error message? sounds good?

Reply:
looks fine to me